### ISSUE TITLE
libxml2 broken?

### ISSUE THREAD

**Issue by Begasus** [haikuports/haikuports#3890]
I've been seeing this a few times already trying to build something where libxml2 is involved:
```
/boot/system/develop/headers/libxml2/libxml/dict.h:23: multiple storage classes in declaration of `xmlDict'
/boot/system/develop/headers/libxml2/libxml/dict.h:24: multiple storage classes in declaration of `xmlDictPtr'
/sources/AbiWord-217fbb622ee64efc181cd67d2742d6a60a086e2d/abi/src/../src/config/abi_rules.mk:243: recipe for target '/sources/AbiWord-217fbb622ee64efc181cd67d2742d6a60a086e2d/abi/src/../src/Haiku_1_x86_OBJ/obj/ut_xml_libxml2.o' failed
```

**Comment by Begasus** [comment]
Not all packages are infected by this, for instance the new PR for fontconfig works ok with libxml2

**Comment by Begasus** [comment]
xmoto for x86_gcc2 is infected by this also, building xmoto_x86 is ok

**Comment by korli** [comment]
seems a header isn't compatible with gcc2.

**Comment by korli** [comment]
fixed in cd615d7aea9d95155ee41d4f3aff400470d128d6

**Comment by Begasus** [comment]
Thanks! Will check later once it's available :)
